.prospector.yml

```diff
@@ -26,4 +26,5 @@ pydocstyle:
         D212,  # Multi-line docstring summary should start at the first line
         D213,  # Multi-line docstring summary should start at the second line
         D404,  # First word of the docstring should not be This
+        W0212,  # protected-access
```

Comment:
Because in the unit tests I test some private functions. this leads to a protected-access warning in the linter.
Nevertheless, the private functions are private for a reason, and still, they need to be tested.

Reply:
Maybe just leave it the linter warnings there and remove it from the disable?